decoder/vaapidecoder_h264.cpp

```diff
@@ -1760,7 +1760,7 @@ YamiStatus VaapiDecoderH264::decodeSlice(NalUnit* nalu)
         status = createPicture(slice, nalu);
         if (status != YAMI_SUCCESS)
             return status;
-        if (!m_currPic
+        if (!m_currPic || !m_prevPic
```

Comment:
Agree with you, but we'd better wait for Linda's unit test. I still wonder we have other issue or not. Another thing we should notice is, the player need fix the key frame issue too, if no, t the seek may take more time then user expected. one extream case is, you have 1-hour video,  but it only has 1 key frame, in this case, the user never sees any video after seek.